rpc/connection.go

```diff
@@ -75,6 +76,14 @@ func (t *connTransport) Dial(context.Context) (Transporter, error) {
 	if err != nil {
 		return nil, err
 	}
+
+	// If the client has requested to disable SIGPIPE, then do so now
```

Comment:
Very minor but I think all the other comments in this file end in punctuation, so would be nice to keep that up here and below.